### PR TITLE
WRN-2730: QA-Sampler: Fix the title is the same as the story name

### PR DESCRIPTION
**Pull request by YunhoLee0**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
Before migrating storybook 6.x, the title of each sample was the same as its story name.
There seems to be a bug when migrating the storybook.

### Resolution
Make the title of each sample the same as the story name.

### Links
WRN-2730

### CODE REVIEW

File: samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js [hunk 1]
```diff
@@ -155,6 +155,8 @@ const StorybookDecorator = (story, config = {}) => {
 		groupId: 'Development'
 	};
 
+	const componentName = config.kind.replace(/^([^/]+)\//, '');
```
Comment (juwonjeong) [inline samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js:158]: I think we can also use regExp in this way.
```suggestion
	const componentName = config.kind.replace(/^\w+\//, '');
```

**Comment by juwonjeong** [comment]
LGTM

**Review by juwonjeong — APPROVED** [review]
LGTM

File: samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js [hunk 2]
```diff
@@ -178,7 +180,7 @@ const StorybookDecorator = (story, config = {}) => {
 	return (
 		<Theme
 			className={classnames(classes)}
-			title={`${config.kind}`.replace(/\//g, ' ').trim()}
+			title={componentName === config.name ? `Sandstone ${componentName}` : `${componentName} ${config.name}`}
```
Comment (MikyungKim) [inline samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js:183]: How about
```suggestion
			title={componentName === config.name ? `${config.kind}`.replace(/\//g, ' ').trim() : `${componentName} ${config.name}`}
```
for other theme usage?

Comment (YunhoLee0) [reply samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js]: I think that's better than before. Thank you!
It's fixed.

**Review by MikyungKim — COMMENTED** [review]
Would you fix this in agate too?

**Review by MikyungKim — APPROVED** [review]
👍 Now we can see the right QA sampler names again. Thanks.

**Pull request merged**